app/marketing/views.py

```diff
@@ -611,6 +624,8 @@ def email_settings(request, key):
         'es': es,
         'msg': msg,
         'navs': settings_navs,
+        'available_languages': ['en', 'pl'],
```

Comment:
You can drop this now that you're doing:` {% get_available_languages as LANGUAGES %}` and `{% get_language_info_list for LANGUAGES as langs %}`